CMakeLists.txt

```diff
@@ -19,6 +19,7 @@ option(
   USE_TELEMETRY
   "Include telemetry functionality in the build. Disabling will exclude all telemetry code from the build."
   ON)
+option(WITH_HYPERSTORE "Include Hyperstore Table Access Method in build" ON)
```

Comment:
Should we just build it always? For simplicity.

Reply:
This is a temporary solution to be able to disable Hyperstore if you discover that it does not work for a particular PG version. This is mostly to deal with changes to PG17 until it has been officially released and we do not see any changes any more. Once PG17 is officially released, this flag is not needed any more.

Reply:
We're at RC1 now, so maybe makes sense to remove it already.